tests/helpers/perf_run_helper.rb

```diff
@@ -787,6 +805,15 @@ def get_perf_result
       perf.log_csv
       copy_archive_files
     end
+    scp_to(master, "util/metrics/collect_metrics_files.rb", "/root/collect_metrics_files.rb")
+    on(master, "ruby /root/collect_metrics_files.rb")
+    start_epoch = File.read("#{@archive_root}/start_epoch")
+    end_epoch = File.read("#{@archive_root}/end_epoch")
+    cmf_output = on(master,
+                    "ruby /root/collect_metrics_files.rb --start_epoch #{start_epoch} --end_epoch #{end_epoch}")
+                 .output
+    filename = cmf_output.match(/\w+-\w+.tar.gz/)[0].to_s
+    scp_from(master, "/root/#{filename}", "#{@archive_root}/#{filename}")
```

Comment:
I am curious as to why we are using `scp_from` here and `archive_file_from` via the `copy_archive_files` method above?

Should this patch be applied to the `copy_archive_files` method instead?  I am unclear as to the difference in context between this additional code and the code within the `if perf` block above.  I don't know where the `stop_monitoring` method comes from or what it does.

Reply:
`stop_monitoring` is from beaker-benchmark:
https://github.com/puppetlabs/beaker-benchmark/blob/8ae906f432ad378e3b59ee16464fa7fcc9b4d9d8/lib/beaker-benchmark/helpers.rb#L81

Reply:
yeah, the whole stop monitoring thing is beaker_benchmark.  And I believe it could fail for it's own reasons... which I didn't want to block the collection of metrics.  I was actually tempted to move this code above the stop monitoring, but the metrics collection is more resilient to being done later if needed than the beaker_benchmark, so I didn't want to take the chance that this code failed and blocked beaker_benchmark from stopping atop...